addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -40,6 +41,27 @@ options.UserValueWidget.include({
     },
 });
 
+Many2oneUserValueWidget.include({
+    /**
+     * @override
+     */
+    async _getSearchDomain() {
+        // We add the current website id to the domain if the model has a
```

Comment:
```suggestion
        // We add the current website's domain if the model has a
```
As it's not just about adding the current website_id in the domain but also the False value.
I feel like it's prone to confusion written like that.
But you can ignore this if you want, it's nitpicking.